server/channels/app/import_functions.go

```diff
@@ -31,9 +31,9 @@ import (
 func (a *App) importScheme(rctx request.CTX, data *imports.SchemeImportData, dryRun bool) *model.AppError {
```

Comment:
Interestingly, when first looking at this issue I figured we have had importers for both resources for a long time which I found quite confusing. 

After a bit of digging, I learnt their purpose was to import data for load-testing needs as the former tool was relying on bulk import format to generate and load the synthetic data (https://github.com/mattermost/mattermost/pull/8928).